src/main/java/Iris/EmptyTaskDescriptorsException.java

```diff
@@ -0,0 +1,17 @@
+package iris;
+
+/**
+ * An exception class representing a situation where the description of a todo task is empty.
+ */
+public class EmptyTaskDescriptorsException extends IrisException {
```

Comment:
I like how you created an EmptyTaskDescriptorsException to avoid repetition in code. Great job!